src/Take.Blip.Builder/InputExpirationHandler.cs

```diff
@@ -106,9 +108,15 @@ public async Task OnFlowProcessedAsync(State state, Message message, Node from,
                     throw new ArgumentException("Message content 'StateId' must be present", nameof(InputExpiration.StateId));
                 }
 
+                if (string.IsNullOrWhiteSpace(inputExpiration?.CurrentSessionState))
```

Comment:
When the version is published, erroes will be raised due to previous jobs...
I think it's better not to throw an error when CurrentSessionState is null